stdlib/CMakeLists.txt

```diff
@@ -107,6 +107,18 @@ swift_create_stdlib_targets("swift-stdlib" "sibopt" TRUE)
 swift_create_stdlib_targets("swift-stdlib" "sibgen" TRUE)
 swift_create_stdlib_targets("swift-test-stdlib" "" FALSE)
 
+foreach(sdk ${SWIFT_SDKS})
+  foreach(arch ${SWIFT_SDK_${sdk}_ARCHITECTURES})
+    set(platform "${SWIFT_SDK_${sdk}_LIB_SUBDIR}")
+
+    add_custom_command(TARGET swift-stdlib-${platform}-${arch}
+      COMMAND
+        "${CMAKE_COMMAND}" -E copy
+        "${SWIFT_SOURCE_DIR}/stdlib/public/legacy_layouts/${platform}/layouts-${arch}.yaml"
+        "${SWIFTLIB_DIR}/${platform}/layouts-${arch}.yaml")
+  endforeach()
+endforeach()
```

Comment:
You need to filter out platforms and architectures that don't have legacy layouts here.

Reply:
Do you want me to duplicate the list or are you OK with a check that the source file exists?

Reply:
I think the source file existing is okay for the CMake side. The compiler will complain if one's missing.